src/unfold/contrib/import_export/templates/admin/import_export/import_form.html

```diff
@@ -4,26 +4,28 @@
     <form action="" method="post" enctype="multipart/form-data">
         {% csrf_token %}
 
-        <p class="bg-blue-50 mb-8 text-blue-500 px-3 py-3 rounded-md text-sm dark:bg-blue-500/20 dark:border-blue-500/10">
+        <div class="bg-blue-50 mb-8 text-blue-500 px-3 py-3 rounded-md text-sm dark:bg-blue-500/20 dark:border-blue-500/10">
```

Comment:
Since `dl` tags can't be inside `p` tag, switched to `div` to capture implemented classes